apps/worker/src/app/workflow/usecases/send-message/send-message.command.ts

```diff
@@ -28,7 +28,7 @@ export class SendMessageCommand extends EnvironmentWithUserCommand {
   notificationId: string;
 
   @IsOptional()
-  _templateId: string;
+  _templateId?: string;
```

Comment:
This was missing and causing erroneous type usage, see above that the same attribute is marked via class-validators as being optional.